influxdb-enterprise/defaults/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+# Role Settings
+influx_node_types: ["meta", "data"]
+influx_node_type: "meta" # By default, install node as a meta-node
+influx_pkg_base_url: "https://s3.amazonaws.com/dl.influxdata.com/enterprise/releases"
+influx_version: "1.2.1-c1.2.2" # The version of the package to install
```

Comment:
@kostasb, I took your suggestion on-board and implemented this. Now both packages are installed at the exact same versions.